### PR TITLE
Feature branch pull request

### PR DESCRIPTION
**Pull request by iammouaz**
### Project requirements

- You should personalize the content of your page. Choose a topic that is different than the one in the original design.
- You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
  - Colors.
  - Typography: font face, size and weight.
  - Layout: composition and space between elements.
- The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.
- You can use Bootstrap for styling, if you want to.
- You should build only these 2 pages:
  - The *home page*.
  - The *about page*.
- Each of these pages should have versions for 2 different screen sizes: 
  - Mobile: up to 768px wide.
  - Desktop: 768px or wider.

- Interactions
  - Links
    - The *home page* should have a link in the menu to the *about page*.
    - The logo in the header links to the *home page*.
  - Mobile menu
    - When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
    - There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design (colors, typography, spacings, etc.).
    - The mobile menu should have a close (X) button that closes the menu.
- Dynamic page
  - The section "Featured speakers" should be created dynamically in JavaScript.
  - You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07).

The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.

### Project documentation

Once you have finished the development of the project, you should record a video presenting the features of the project you built. It is a video with a **maximum length of 5 minutes**. The content of the video should include:

- A description of the project.
- A demo of the project features (different pages and different screen sizes).
- You should also highlight some interesting piece of code or something you built that you are very proud of.

For recording the video you can use tools like [Loom](https://www.loom.com/) that let you share a private link to the recording, and configure a shot that shows your computer screen and your face at the same time in a small picture.

**Add the video link to your pull request description.**

[Loom Video For my project ](https://www.loom.com/share/e6ebc117582e4497a8b65a569601aaf2)

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  @iammouaz 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        <li class="top-nav-list">Join</li>
+        <li class="top-nav-list">Sponsor</li>
+        <li class="top-nav-list">News</li>
+       </ul>
+    </div>
+<div class="home-contanier">
+           <h1 class="home-title">Space Tech Expo 2021</h1>
+
+           <h2 class="home-haeder">SPACE TECH EXPO EUROPE RETURNS FOR 2021</h2>
+
```
Comment (mikenath223) [inline index.html:67]: - [x] You have text flowing into other text, kindly fix it.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    </div>
+<div class="home-contanier">
+           <h1 class="home-title">Space Tech Expo 2021</h1>
+
+           <h2 class="home-haeder about">SPACE TECH EXPO EUROPE RETURNS FOR 2021 </h2>
+<div class="red-line"></div>
+           <p class="home-p">
+                The world-leading IoT event series includes a North America virtual event, plus two physical events in London and Amsterdam throughout 2021.
+
+                The events will bring together key industries from across the globe for two days of top-level content and discussion across 4 co-located events covering IoT,  cyber security, cloud, blockchain, AI and big data.
```
Comment (mikenath223) [inline about.html]: - [x] Also, fix the text flowing into other content on the about page.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+ <button type="button" class="main-btn text-white">Join Space Tech Expo 2021</button>
+    </section>  
+
+    <section class="speakers">
+<h2 class="speaker-main">Featured Speakers</h2>
+<hr class="red-line">
+<div id="speakers" class="card-all"></div>
+    </section>
+
```
Comment (mikenath223) [inline index.html]: - [x] On the featured speakers images kindly add the background behind each image as seen in the original.

| Original | Yours |
| ------- | ----- |
|  |

File: index.html [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    <div class="sponser3"></div>
+    <div class="sponser4"></div>
+    <div class="sponser5"></div>
+</div>
+    </section>
+    <footer class="end">
+        <img class="footer-logo" src="./img/footer-logo.png" alt="Logo">
+        <p class="footer-p">2021 Creative Commons Germany.<br> Some Rights Reserved.</p>
+    </footer>
+    <script src="./js/main.js"></script>
```
Comment (mikenath223) [inline index.html:115]: - [x] On the mobile screen the footer of the home page is expected to look like the given design with just the `More` text having a white background.

| Original | Yours |
| ------- | ----- |
|  |

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @iammouaz  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**